src/components/editor/actions/CloseButton.jsx

```diff
@@ -14,7 +14,7 @@ const CloseButton = (props) => {
   const history = useHistory()
 
   const resourceKey = useSelector(state => currentResourceKey(state))
-  const resourceHasChanged = useSelector(state => resourceHasChangesSinceLastSave(state))
+  const resourceHasChanged = useSelector(async state => await resourceHasChangesSinceLastSave(state))
```

Comment:
Looking at the related test failure, this is likely not doing what I had hoped and I'm not inclined to go down this rabbit hole.